pytensor/graph/rewriting/utils.py

```diff
@@ -44,35 +44,36 @@ def rewrite_graph(
     """
     from pytensor.compile import optdb
 
-    return_fgraph = False
     if isinstance(graph, FunctionGraph):
         fgraph = graph
-        return_fgraph = True
+    elif isinstance(graph, Variable):
+        fgraph = FunctionGraph(outputs=[graph], clone=clone)
     else:
-        if isinstance(graph, list | tuple):
-            outputs = graph
-        else:
-            assert isinstance(graph, Variable)
-            outputs = [graph]
-
-        fgraph = FunctionGraph(outputs=outputs, clone=clone)
+        fgraph = FunctionGraph(outputs=graph, clone=clone)
```

Comment:
Slighly prefer the previous branching logic

Reply:
What did you prefer about it?
I had two objectives here: remove the `return_fgraph` variable, and flatten the logic (one level of conditions rather than two nested levels). Does one of these things bothers you more than the other?

Reply:
I don't like the two calls to `FunctionGraph`. I would prefer first convert a variable to a list and then do a single call to FunctionGraph?

Reply:
Ah ok thanks, I'll try to do something like that

Reply:
How about now?

Reply:
Looks great